contracts/AnchorRepository.sol

```diff
@@ -192,19 +172,13 @@ contract AnchorRepository is Initializable {
   // @param _signature Signed message
   function isSignatureValid(
```

Comment:
In the upcoming PR's we will remove this function, right?

Reply:
I am thinking about it right now. @lucasvo mentioned that we should keep it because we might need it but  it does not make sense if we do not have a purpose associated with it. 725 does not have a purpose for that and we plan to have generated ints based on a namespace in order to avoid collisions. So yea, we will remove it.